src/core/lender/distributor/trad_distributor.sol

```diff
@@ -27,20 +27,32 @@ contract TraditionalDistributor is Distributor {
     // ERC20
```

Comment:
On naming:
"TraditionalDistributor" is not really helpful. Is this a switchabledistirbutor (because it can be set to 'poolClosing'?